### PR TITLE
Update sentence structure in line 290 of readme.md

### PR DESCRIPTION
**Pull request by capndave**
## Description

I made a couple of changes to improve readability on line 290 of `readme.md`.

1. changed "one exceptions" to "one exception"
2. Broke up long sentence with multiple colons into two sentences, elaborated a bit about inability to use braces in JSON5 format to hopefully make things more clear

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -287,8 +287,8 @@ cli input.txt --setting 'golf: false, hotel-india: ["juliet", 1]'
 Configuration for the parser and compiler of the processor.
 Can be set from [configuration files][config-file].
 
-The given settings are [JSON5][], with one exceptions: surrounding braces must
-not be used: `"foo": 1, "bar": "baz"` is valid
+The given settings are [JSON5][], with one exception: surrounding braces must
+not be used.  Instead, simply use JSON syntax without braces, i.e. `"foo": 1, "bar": "baz"`.
```
Comment (wooorm) [inline readme.md]:
```suggestion
not be used.
So, use JSON syntax without braces, such as `"foo": 1, "bar": "baz"`.
```

**Review by wooorm — COMMENTED** [review]
I like it a lot, thanks, those were definitely bugs.

I do suggest not using `simply`, because it might not be simple for some readers, and not to use `i.e.`, as readers are often confused between `e.g.,` and `i.e.,`, I prefer `such as`, or `as in` instead.

File: readme.md (hunk 1 above)
Comment (capndave) [reply readme.md]: Sounds good! Updated and pushed

**Review by ChristianMurphy — APPROVED** [review]

**Review by wooorm — APPROVED** [review]

**Comment by wooorm** [comment]
Thanks @capndave!

**Pull request merged**